BUFR_TableD_en_09.csv

```diff
@@ -216,6 +216,35 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,031001,Delayed replication factor,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,002090,Instrument wavelength,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,115000,Delayed replication of 15 descriptors,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,031002,Extended delayed replication factor,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,007007,Height,,(see Note 15),Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,301021,Latitude/longitude (high accuracy),= 2 Time averaged,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015073,Attenuated backscatter,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015064,Uncertainty in attenuated backscatter,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015074,Particle backscatter coefficient,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015066,Uncertainty in particle backscatter coefficient,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015075,Particle extinction coefficient,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015068,Uncertainty in particle extinction coefficient,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015076,Particle lidar ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015077,Uncertainty in lidar ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015078,Particle depolarization ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,015072,Uncertainty in depolarization ratio,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,033002,Quality information,,(19) The Quality information descriptor 0 33 002 relates to all parameters provided in the Backscatter data section and is given for each respective vertical bin of the profile measurement.,Operational
```

Comment:
@amilan17 Here "(19)" to be replaced by "(see Note 19)"

Reply:
done